src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb

```diff
@@ -831,6 +827,15 @@ Public Class BuildDevDivInsertionFiles
                     Continue For
                 End If
 
+                If parent.EndsWith("DevDivPackages\Roslyn", StringComparison.OrdinalIgnoreCase) Then
+                    Continue For
+                End If
+
+                ' Wild cards aren't supported
+                If item.Contains("*") Then
+                    Continue For
```

Comment:
Is this just to prevent somebody from trying to add *? If so, why not throw an exception?